### ISSUE TITLE
[V1.1] Contract & Subgraph deployment

### ISSUE THREAD

**Issue by sanchaymittal** [connext/monorepo#5031]
#### **_@connext/smart-contracts_**
- [ ] Deploy Op Roots v1.1 contract once DW completes the missing stuff.
- [ ] Add a feature to clear/set hub domain on RootManager in init script if needed.

**_@connext/nxtp-subgraph_**
1. @connext/nxtp-subgraph/amarok-hub-staging
- [ ] Add a new event: `AggregateRootSavedOptimistic` to record valid aggregated roots. 
- [ ] 
2. @connext/nxtp-subgraph/amarok-runtime-staging
### Linked Issues & Documentation
https://www.notion.so/Offchain-spec-for-OPT-roots-V1-1-0fdefbab648e426aabaa96b0eabe232e?pvs=4#5aafc736ffde457183e680118029a58a

**Comment by preethamr** [comment]
@wanglonghong @liu-zhipeng 

Splitting this task, so that we can go in parallel over timezones, into these tasks:
https://github.com/connext/monorepo/issues/5032
https://github.com/connext/monorepo/issues/4994
https://github.com/connext/monorepo/issues/4995

**Comment by wanglonghong** [comment]
duplicates of https://github.com/connext/monorepo/issues/5032 and https://github.com/connext/monorepo/issues/4994
